hail/src/test/scala/is/hail/expr/ir/IRSuite.scala

```diff
@@ -2244,6 +2244,14 @@ class IRSuite extends HailSuite {
     assertEvalsTo(makeNDArrayRef(matMulCube, IndexedSeq(0, 0, 0)), 30.0)
   }
 
+  @Test def testNDArrayInv() {
+    implicit val execStrats: Set[ExecStrategy] = ExecStrategy.compileOnly
```

Comment:
It's not clear to me if this is an oversight, but most of our NDArray tests are not actually tested, because execStrats are Set(); the values specified in these assertEvalsTo are arbitrary. See TestNDArrayMatMul as a worrisome example of this.

Reply:
oof, good catch! That's consistent with my memory of the history here. Will happily (and a bit nervously...) review the PR to reenable most tests.

These nodes are also tested pretty well in Python, so this isn't as horrifying as it may appear at first.

Reply:
Yeah, I verified that matmul for instance works perfectly enabled.